src/Message/Launch/Validator/AbstractLaunchValidator.php

```diff
@@ -78,4 +84,14 @@ protected function reset(): self
 
         return $this;
     }
+
+    public function isStateValidationRequired(): bool
```

Comment:
why public?

Reply:
The client of the class is easily able to confirm is the validations are enabled or not